e2e/data-factory/utils/api-client.ts

```
@@ -0,0 +1,137 @@
+    
+    body(data: any): RequestBuilder {
+        this.bodyData = data;
+        return this;
+    }
+    
+    expect(status: number): RequestBuilder {
+        this.expectedStatus = status;
+        return this;
+    }
+    
+    async then(resolve: (value: any) => void, reject?: (error: any) => void): Promise<void> {
```

Biome (2.1.2)

[error] Do not add then to a class.



(lint/suspicious/noThenProperty)